spec/API_specification/signatures/creation_functions.py

```diff
@@ -237,6 +231,12 @@ def linspace(start: Union[int, float], stop: Union[int, float], /, num: int, *,
     -------
     out: array
         a one-dimensional array containing evenly spaced values.
+
```

Comment:
Locally this doesn't render correctly:

![Screenshot from 2022-04-22 11-43-29](https://user-images.githubusercontent.com/8246949/164700337-750ad046-a113-4d6e-9d74-ef91f9bd144f.png)

But another newline seems to does the trick:

```suggestion


```